frontend/scripts/verify-e04-04.sh

```
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+# E04-04: ディレクトリ構成とパスエイリアスの検証スクリプト
+# Issue #13 の受入基準（AC）を検証する
+# 実行方法: ./scripts/verify-e04-04.sh
+
+# スクリプトのディレクトリの親（frontendルート）に移動
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)